src/business_logic/transaction/objects/internal_invoke_function.rs

```diff
@@ -212,6 +217,57 @@ impl InternalInvokeFunction {
             );
         Ok(transaction_execution_info)
     }
+
+    fn _charge_fee<S>(
```

Comment:
add an annotation to prevent the clippy warning and remove the `_` please

Reply:
we can create a trait to avoid duplicating logic here

Reply:
fixed, we can do the trait in a different PR as it involves no minor changes.